migrate/loader.go

```diff
@@ -0,0 +1,137 @@
+package migrate
+
+import (
+	"fmt"
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+	"os"
+	"strings"
+)
+
+var loaderCommand = &cobra.Command{
+	Use:   "loader",
+	Short: "Migrate your loader versions to newer versions.",
+	Args:  cobra.ExactArgs(1),
```

Comment:
Please document the arguments to this command (even just something in the `Use` string would be much better)